dist/spec.yaml

```
@@ -3655,6 +3682,30 @@ paths:
       summary: Deactivate Identity Provider
       tags:
         - IdentityProvider
+  '/api/v1/idps/{idpId}/metadata.xml':
+    get:
+      consumes:
+        - application/json
+      description: Federation metadata document for the IdP
+      operationId: identityProviderMetadata
+      parameters:
+        - in: path
+          name: idpId
+          required: true
+          type: string
+      produces:
+        - application/octet-stream
+      responses:
+        '200':
+          description: Success
+          schema:
+            items:
+              type: integer
```

According to Swagger compiler in Java this should be something like
```suggestion
          schema:
            items:
              $ref: '#/definitions/{new_data_type}'
```
Also I am not sure that this request will return Integer[]

---

Are you sure this is the syntax for OpenAPI v2?  This seems odd, we have `integer` other places in the spec already